addons/web/controllers/main.py

```diff
@@ -1710,7 +1710,8 @@ def get_fields(self, model, prefix='', parent_name= '',
                       'value': val, 'children': False,
                       'field_type': field.get('type'),
                       'required': field.get('required'),
-                      'relation_field': field.get('relation_field')}
+                      'relation_field': field.get('relation_field'),
+                      'default_export': model == 'res.partner' and import_compat and field_name == 'name'}
```

Comment:
We talk about a ``force_default=True`` parameter on the field definition itself. Didn't you try it ? It would be more modular.

Reply:
Sorry I misunderstood, it is done now with the parameter default_export_for_import on field definition. I have not used the name force_default as the goal is to force only the field for exportation in import compatible mode (in standard mode, display_name is already present which is removed in import compatible mode).